SU-SOE/stanford_bean_types
# 7.x 2.x update to postcard linked

# READY FOR REVIEW

# Summary
- This is an upstream merge with the _Stanford Bean Types_ 7.x-2.x on the SU-SWS repo.
- This [captures changes to the Social Media block](https://stanfordits.atlassian.net/browse/SOE-2368). 
- This is needed so we can use the _Alt Tag Check_ module


# Needed By (12/7/2017)
- Sprint end

# Criticality
- Needed to put the alt-tags module on the site
- Fixes broken stuff

# Steps to Test
- If necessary, clone _Stanford Bean Types_
`git clone git@github.com:SU-SOE/stanford_bean_types.git`
- Switch to this branch (
`git checkout 7.x-2.x-update-to-postcard-linked`
- Clear cache
`drush cc all `


## Verify _Social Media Bean Types_
 - Navigate to the **homepage** and verify the social media block looks like:
![screen shot 2017-12-06 at 16 16 20](https://user-images.githubusercontent.com/284440/33692193-dae10110-daa0-11e7-9626-1695f8681782.png)
- Navigate to **admin/structure/features/stanford_bean_types/diff/field_instance** and verify that the only thing overridden is default_image for _Stanford Bean Types_.

![screen shot 2017-12-06 at 17 01 27](https://user-images.githubusercontent.com/284440/33693361-5fb632e2-daa7-11e7-8d7c-48273c065acb.png)

## Verify _Postcard Linked_
- Create a postcard linked block (**block/add/stanford-postcard-linked**)
- Place a your block on a page using either context or blocks page.
- Verify your block's image and title are functioning links.

## Verify _Alt Tag Check_ module
- `git clone git@github.com:SU-SWS/stanford_alt_check.git`
- `drush cc all`
- `drush en stanford_alt_check`
- Check out lastest tag
`git checkout 7.x-1.1-alpha1`
- `drush cc  all`
- Navigate to **/admin/content/images**
- Click on **Rescan Images** (then go make coffee)
- Verify it looks like this:
![screen shot 2017-12-06 at 16 47 20](https://user-images.githubusercontent.com/284440/33693100-9ecd0336-daa5-11e7-9881-c4ea3cd1492c.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2450
https://stanfordits.atlassian.net/browse/SOE-2368


## Related PRs

## More Information


## Folks to notify
@kbrownell,
I apologize for the tardiness. I needed to rebuild my local environment and ran into issues with that.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
